src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -338,32 +349,26 @@ export class ResponsiveTabs implements IResponsiveComponent {
     );
   }
 
-  private getTabsInTabSection(): HTMLElement[] {
-    let tabsInSection = [];
-    _.each(this.tabSection.el.children, childElement => {
-      if (Utils.isHtmlElement(childElement)) {
-        let child = $$(<HTMLElement>childElement);
-        if (!child.hasClass('coveo-tab-dropdown') && child.hasClass(Component.computeCssClassNameForType(this.ID))) {
-          tabsInSection.push(child.el);
-        }
-      }
-    });
-    return tabsInSection;
-  }
+  private fromDropdownToTabSection(tab: Dom) {
+    const lastTabInTabSection = last(this.tabsInTabSection);
+    if (!lastTabInTabSection) {
+      this.tabSection.prepend(tab.el);
+    }
+
+    const comesAfterInInitialTabsOrder = this.initialTabsOrder.indexOf(tab.el) > this.initialTabsOrder.indexOf(lastTabInTabSection);
```

Comment:
I'm finding it difficult to understand this statement. I wonder if there are certain arrangements of tabs where it would not work? Especially since the lastTab is changing as the `while` loop progresses. Couldn't we take the initial tab order, loop through and append?